internal/cmd/local/command_kafka_start.go

```diff
@@ -42,6 +43,11 @@ func (c *Command) newKafkaStartCommand() *cobra.Command {
 }
 
 func (c *Command) kafkaStart(cmd *cobra.Command, args []string) error {
+	err := checkMachineArch()
```

Comment:
I would put this inside of `if runtime.GOOS != "windows" { }` since `uname` doesn't work on Windows